tests/integration_tests/security/spectre-mitigations

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Checks SPECTREv2 mitigations are present.
+
+arch=`uname -m`
+
+if [ ${arch} = "x86_64" ]; then
+    # Validate SPECTRE V2 mitigations are enabled:
+    # - `IBPB: conditional` 
+    # - `eIBRS` (`Enhanced IBRS` intel, `Retpolines` AMD).
+    BODY=$(sudo cat /sys/devices/system/cpu/vulnerabilities/spectre_v2)
+    if [[ $BODY =~ "IBPB: conditional" ]] && [[ $BODY =~ "Enhanced IBRS" || $BODY =~ "Retpolines" ]]; then
```

Comment:
A comment above was saying that IBPB needs to be at least conditional. Here, we only check *only* for conditional.

Reply:
Updated to `if [[ $BODY =~ "IBPB: conditional" || $BODY =~ "IBPB: always-on" ]] && [[ $BODY =~ "Enhanced IBRS" || $BODY =~ "Retpolines" ]]`